src/index.js

```diff
@@ -0,0 +1,40 @@
+document.getElementById("up-button").onclick = function(){
```

Comment:
Prefer using `addEventListener` to register event handlers on an element rather than assigning to the on* event attributes.

```js
document.getElementById("up-button").addEventListener("click", function(){
  ...
});
```